src/components/BondingCurve.vue

```diff
@@ -0,0 +1,158 @@
+<script>
+import { Scatter } from 'vue-chartjs';
+
+export default {
+  name: 'BondingCurve',
+  extends: Scatter,
+  props: {
+    data: {
+      type: Object,
+      default: null,
+    },
+  },
+  mounted() {
+    this.renderChart(this.chartData, this.chartOptions);
+  },
+  computed: {
+    chartData() {
+      const sellPrice = this.data.sellPrice / 1000000000000000000;
+      const buyPrice = this.data.buyPrice / 1000000000000000000;
+      const supply = this.data.totalSupply / 1000000000000000000;
```

Comment:
these values can overrun js number precision, we should use bignumber for such calculations